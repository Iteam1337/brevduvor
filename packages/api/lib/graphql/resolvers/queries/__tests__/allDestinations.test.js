const { ApolloServer } = require('apollo-server-express')
const { createTestClient } = require('apollo-server-testing')
const { gql } = require('apollo-server-express')
const { allDestinations } = require('./../index')
const { serverConfig, osrmInstance } = require('./../../../../../index')
const { query } = createTestClient(new ApolloServer(serverConfig))

describe('allDestinations', () => {
  it('exists', () => {
    expect(allDestinations).toBeDefined()
  })

  it('matches snapshot', async () => {
    const ALL_DEST = gql`
      query {
        allDestinations {
          alias
          lat
          lon
        }
      }
    `

    const res = await query({ query: ALL_DEST })

    expect(res.data).toMatchSnapshot()
  })
})
