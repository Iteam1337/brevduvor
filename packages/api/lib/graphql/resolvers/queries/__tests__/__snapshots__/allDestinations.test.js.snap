// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allDestinations matches snapshot 1`] = `
Object {
  "allDestinations": Array [
    Object {
      "alias": "Storuman",
      "lat": 65.090833,
      "lon": 17.1075,
    },
    Object {
      "alias": "Kvikkjokk",
      "lat": 66.9501067,
      "lon": 17.70861,
    },
    Object {
      "alias": "Slussfors",
      "lat": 65.4308046,
      "lon": 16.2481741,
    },
  ],
}
`;
